src/Transports.AspNetCore.SystemTextJson/GraphQLBuilderSystemTextJsonExtensions.cs

```diff
@@ -25,6 +26,7 @@ public static class GraphQLBuilderSystemTextJsonExtensions
         /// Affects JSON returned by the middleware.
         /// </param>
         /// <returns>GraphQL Builder.</returns>
+        [Obsolete]
```

Comment:
See naming convention here - `GraphQLBuilderSystemTextJsonExtensions`. That's what I talked about in https://github.com/graphql-dotnet/graphql-dotnet/issues/2790